cobc/ChangeLog

```diff
@@ -1,4 +1,24 @@
 
+2022-07-04  Nicolas Berthier <nicolas.berthier@ocamlpro.com>
```

Comment:
I think somewhere here should be a reference to the FRs 29 and 130 (also "extended as `[feature-requests:#29], [feature-requests:#130]` in the svn commit message).